### PR TITLE
init user model

### PR DESCRIPTION
**Pull request by duchk-2510**
Generate user model
Write validation for user model
Write seed for sample data
image

### CODE REVIEW

**Comment by duchk-2510** [comment]
ready

File: app/helpers/session_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module SessionHelper
```
Comment (vanvtt-0952) [inline app/helpers/session_helper.rb]: Xóa file

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
```
Comment (vanvtt-0952) [inline config/routes.rb:4]: hai line code này đặt sau line code root nhé

**Comment by duchk-2510** [comment]
ready

File: db/migrate/20210512025724_create_delivery_infomations.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      t.string :receipiant_phone
+      t.string :receipiant_address
+      t.bigint :user_id, null: false
+
+      t.timestamps
+    end
+    add_foreign_key :shipments, :users
+  end
+end
+20210512025854
```
Comment (vanvtt-0952) [inline db/migrate/20210512025724_create_delivery_infomations.rb]: j dây cậu

File: db/schema.rb [hunk 4]
```diff
@@ -10,6 +10,82 @@
...
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["user_id"], name: "fk_rails_71a7b88fec"
+  end
+
+  create_table "users", charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+    t.string "name"
+    t.string "address"
+    t.string "phone"
+    t.string "email", null: false
+    t.integer "admin"
```
Comment (vanvtt-0952) [inline db/schema.rb]: `"admin"` -> `role` chứ cậu

Comment (duchk-2510) [reply db/schema.rb]: à dạ

**Comment by duchk-2510** [comment]
ready

**Pull request merged**